worlds/alttp/docs/multiworld_en.md

```diff
@@ -8,8 +8,8 @@
 - SNI is not compatible with (Q)Usb2Snes.
 - Hardware or software capable of loading and playing SNES ROM files
```

Comment:
slight grammar suggestion. I use bsnes-plus for lttp, so also added that here if you'd like it. installation process to get it running is even less than snes9x-nwa. the networking is always enabled so it just works as is, if you want to add a blurb for it below.
```suggestion
- Hardware or software capable of loading and playing SNES ROM files, including:
    - An emulator capable of connecting to SNI
      ([snes9x rr](https://github.com/gocha/snes9x-rr/releases),
       [BizHawk](https://tasvideos.org/BizHawk), or
      ([snes9x-nwa](https://github.com/Skarsnik/snes9x-emunwa/releases), [snes9x rr](https://github.com/gocha/snes9x-rr/releases),
       [BSNES-plus](https://github.com/black-sliver/bsnes-plus),
       [BizHawk](http://tasvideos.org/BizHawk.html), or
       [RetroArch](https://retroarch.com?page=platforms) 1.10.1 or newer)
    - An SD2SNES, [FXPak Pro](https://krikzz.com/store/home/54-fxpak-pro.html), or other compatible hardware. **note: 
modded SNES minis are currently not supported by SNI. Some users have claimed success with QUsb2Snes for this system,
but it is not supported.**
```